pgrx-examples/decoder/src/lib.rs

```diff
@@ -240,10 +240,8 @@ unsafe extern "C" fn pg_decode_begin_txn(
     ctx_ptr: *mut pg_sys::LogicalDecodingContext,
     _txn_ptr: *mut pg_sys::ReorderBufferTXN
 ){
-    let ctx = unsafe { PgBox::from_pg(ctx_ptr) } ;
-    let mut state = unsafe {
-        PgBox::from_pg(ctx.output_plugin_private as *mut DecodingState)
-    };
+    let ctx = PgBox::from_pg(ctx_ptr);
+    let mut state = PgBox::from_pg(ctx.output_plugin_private as *mut DecodingState);
```

Comment:
You're gonna want to revert this commit, because this should have `deny(unsafe_op_in_unsafe_fn)`.